processor/parser_report.py

```diff
@@ -18,7 +31,7 @@ def __init__(self, logger):
         self._logger = logger
         self._is_israel = None
 
-    def parse_file(self, file_path):
+    def parse_file(self, file_path) -> Tuple[str, dict]:
```

Comment:
If we going to add type-hinting we need to do this over all the project, not a specific place.